bin/wasm-node/javascript/README.md

```diff
@@ -19,10 +19,10 @@ const chain_spec = Buffer.from(fs.readFileSync('./westend.json')).toString('utf8
 smoldot
   .start({
     chain_spec: chain_spec,
-    json_rpc_callback: (resp) => {
+    json_rpc_callback: (resp, chain_index) => {
         // Called whenever the client emits a response to a JSON-RPC request,
         // or a JSON-RPC pub-sub notification.
-        console.log(resp)
+        console.log(resp, chain_index)
     }
   })
```

Comment:
Well, you need to update the README to pass the `chain_index` when sending the request as well